simplq/src/components/pages/Join/Form.jsx

```diff
@@ -98,7 +105,7 @@ export function JoinQueueForm(props) {
         onChange={handleContactChange}
         onKeyDown={(e) => handleEnterPress(e, handleClick)}
       />
-      <JoinButton />
+      <ActionButton />
```

Comment:
I'd prefer calling this "CreateTokenButton"

Reply:
Sure